server/basicServices/basicServices/BasicServicesService.js

```diff
@@ -720,7 +720,13 @@ exports.registerYourself = async function (body, user, xCorrelator, traceIndicat
     }
 
     // update old release configuration
-    let beaqueathYourDataAndDieForwardingName = "PromptForEmbeddingCausesRequestForBequeathingData";
+    let beaqueathYourDataAndDieForwardingName
+    if(forwardingName == undefined){
+      beaqueathYourDataAndDieForwardingName = "PromptForEmbeddingCausesRequestForBequeathingData";
+    }else{
+      beaqueathYourDataAndDieForwardingName = "PromptForEmbeddingInitiatesEmbeddingProcess.RequestForBequeathingData"
```

Comment:
if forwardingName is not undefined , then you have to assign the incoming "forwardingName" , not again hardcoding another forwarding name here.

Reply:
done